omod/src/main/webapp/fragments/field/maritalStatus.gsp

```diff
@@ -6,8 +6,8 @@
     <select name="${config.formFieldName}">
```

Comment:
@ssmusoke  Is this coding standard to other HMIS forms other than the ART card??